kinto_amo/views/services.py

```diff
@@ -1,21 +1,79 @@
-from cliquet import Service
+from cliquet import Service, utils
+from cliquet.storage import Filter
 
+from kinto2xml.exporter import (
+    write_addons_items, write_plugin_items, write_gfx_items, write_cert_items
+)
+from lxml import etree
 
 path = '/blocklist/{api_ver}/{application_guid}/{application_ver}/'
 
+PARENT_PATTERN = "/buckets/{bucket}/collections/{collection}"
+
 blocklist = Service(name="blocklist", path=path,
                     description="Blocklist data")
 
 
 @blocklist.get()
 def get_blocklist(request):
-    api_ver = request.matchdict['api_ver']
+    api_ver = int(request.matchdict['api_ver'])
     app = request.matchdict['application_guid']
     app_ver = request.matchdict['application_ver']
-    query = request.body
 
-    return {
-        'API version': api_ver,
-        'Application': app,
-        'Application version': app_ver,
-        'Query': query}
+    last_update = 0
+    resources = request.registry.amo_resources
+
+    # Addons blocklist
+    addons_records, addons_records_count = request.registry.storage.get_all(
+        collection_id="record",
+        parent_id=PARENT_PATTERN.format(**resources['addons']),
+        filters=[Filter('enabled', True, utils.COMPARISON.EQ)])
```

Comment:
what about a small helper `get_records(request, 'addons')` instead of duplicating ?